linera-views/src/views/map_view.rs

```diff
@@ -776,7 +776,7 @@ where
     /// assert_eq!(count, 1);
     /// # })
     /// ```
-    pub(crate) async fn for_each_key_value_or_bytes<'a, F>(
+    pub async fn for_each_key_value_or_bytes<'a, F>(
```

Comment:
So you want to make it public after all? I'd keep this and the `ValueOrBytes` type private.

Reply:
Yes please